src/gui/folderman.cpp

```diff
@@ -849,7 +849,7 @@ Folder *FolderMan::folderForPath(const QString &path)
     foreach(Folder* folder, this->map().values()) {
         const QString folderPath = folder->cleanPath()+QLatin1Char('/');
 
-        if(absolutePath.startsWith(folderPath)) {
+        if(absolutePath.startsWith(folderPath, Qt::CaseInsensitive)) {
```

Comment:
I think this should rather be 
```
startsWith(folderPath, Utility::isLinux() || Utility::isBSD() ? Qt::CaseSensitive : Qt::CaseInsensitive)) {
..
```
because in case of case sensitive file system, we want to compare case sensitive.